src/lib.rs

```diff
@@ -291,9 +305,7 @@ impl MyDrone {
         if let Some(channel) = self.packet_send.get(&dest) {
             // packet drop logic
             if matches!(packet.pack_type, PacketType::MsgFragment(_)) {
-                // TODO: fix rand::rng();
-                let mut rng = rand::rng();
-                let random_number: f32 = rng.random_range(0.0..=1.0);
+                let random_number: f32 = generate_random_value_in_range(0.0..=1.0);
 
                 if random_number < self.pdr {
                     self.make_and_send_nack(
```

Comment:
At this point we could also make the function have all the drop logic inside and just return a bool that tells you if we are dropping the packet

Reply:
Good, I'll add it